guide-app-monitoring-ocp4.2.md

```diff
@@ -0,0 +1,132 @@
+---
+permalink: /guides/app-monitoring-ocp4.2/
+layout: guide-markdown
+title: Application Monitoring on Red Hat OpenShift Container Platform (RHOCP) 4.2 with Prometheus and Grafana
+duration: 60 minutes
+releasedate: 2020-02-02
+description: Learn how to monitor applications on RHOCP 4.2 with Prometheus and Grafana.
+tags: ['monitoring', 'prometheus', 'grafana']
+guide-category: monitoring
+---
+
+__This guide has been tested with RHOCP 4.2/Kabanero 0.3.0.__
+
+
+For application monitoring on RHOCP, you need to set up your own Prometheus and Grafana deployments. Both Prometheus and Grafana can be set up via Operator Lifecycle Manager (OLM).
+
+## Prerequisites
+
+Prior to deploying Prometheus, ensure that there is a running application that has a service endpoint for outputting metrics in Prometheus format.
+
+It is assumed such a running application has been deployed to the RHOCP cluster inside a project/namespace called `myapp`, and that the Prometheus metrics endpoint is exposed on path `/metrics`.
+
+Git clone this repo to get going right away:
```

Comment:
@jantley-ibm - would we need to include this if we are now linking to files?

Reply:
@SueChaplain I'm OK with removing it; will make this change